src/test/java/net/sf/jabref/logic/formatter/bibtexfields/HtmlToUnicodeFormatterTest.java

```diff
@@ -0,0 +1,59 @@
+package net.sf.jabref.logic.formatter.bibtexfields;
+
+import net.sf.jabref.Globals;
+import net.sf.jabref.JabRefPreferences;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class HtmlToUnicodeFormatterTest {
+
+    private final HtmlToUnicodeFormatter formatter = new HtmlToUnicodeFormatter();
+
+    @Before
+    public void setUp() throws Exception {
+        Globals.prefs = JabRefPreferences.getInstance();
```

Comment:
You are correct. That comes from a copy paste from the other formatter tests, which all share this problem. I will fix all of them in this PR.